aws/aws.go

```diff
@@ -942,6 +943,42 @@ func (s *awsOps) detachInternal(volumeID, instanceName string, options map[strin
 	return err
 }
 
+func isErrorModificationNotFound(err error) bool {
+	return strings.HasPrefix(err.Error(), awsErrorModificationNotFound)
+}
+
+func (s *awsOps) IsVolumesReadyToExpand(volumeIDs []*string) (bool, error) {
+	modificationStateRequest := &ec2.DescribeVolumesModificationsInput{
+		VolumeIds: volumeIDs,
+	}
+	describeOutput, err := s.ec2.DescribeVolumesModifications(modificationStateRequest)
+	states := describeOutput.VolumesModifications
+	if err != nil {
+		// modification state not found, this indicates no change has occurred before.
+		if isErrorModificationNotFound(err) {
+			return true, nil
+		}
+		return false, fmt.Errorf("unable to get volumes' modification states: %v", err)
```

Comment:
nit:
"unable to get volumes' modification states" -> "unable to get modification states for aws volumes"